app/javascript/components/layout/Footer.jsx

```diff
@@ -51,29 +41,24 @@ const Footer = () => (
                     Slack Guidelines
                 </a>
             </div>
-
             <div className="footer-col">
-                <p className="footer-col-item footer-col-title">For Sponsors</p>
-                <a className="footer-col-item" href="#">
-                    Sponsor Us
+                <p className="footer-col-item footer-col-title">Resources</p>
+                <a className="footer-col-item" href="/jobs">
+                    Job Board
                 </a>
             </div>
         </div>
 
         <div className="footer-right">
             <div className="social-links">
-                <a href="#">
+                <a href="mailto:organizers@wnb-rb.dev" target="_blank" rel="noopener noreferrer">
                     <Email className="social-icon" />
                 </a>
                 <a href="https://twitter.com/wnb_rb" target="_blank" rel="noopener noreferrer">
                     <Twitter className="social-icon" />
                 </a>
-                <a
-                    href="https://www.linkedin.com/company/wnb-rb"
-                    target="_blank"
-                    rel="noopener noreferrer"
-                >
-                    <LinkedIn className="social-icon" />
+                <a href="https://github.com/wnbrb" target="_blank" rel="noopener noreferrer">
+                    <Github className="social-icon" />
```

Comment:
Replaced our LinkedIn icon with GitHub because we're not super active on LinkedIn and our GitHub is an awesome reflection of the work our community is doing!